backend/de.metas.business/src/main/java/de/metas/order/model/interceptor/C_Order.java

```diff
@@ -238,17 +238,39 @@ public void setIncoterms(final I_C_Order order)
 		}
 
 		final int c_Incoterms;
+		final String incotrmLocation;
 
 		if (order.isSOTrx())
 		{
-			c_Incoterms = bpartner.getC_Incoterms_Customer_ID();
+			if (bpartner.getC_Incoterms_Customer_ID() > 0)
+			{
+				c_Incoterms = bpartner.getC_Incoterms_Customer_ID();
+				incotrmLocation = bpartner.getIncotermLocation();
+			}
+			else
+			{
+				c_Incoterms = bpartner.getC_BP_Group().getC_Incoterms_ID();
```

Comment:
If it's not in the partner, just take the default. I see no point in taking it from the group if it was not in the partner.